pkg/event_processor/iworker.go

```diff
@@ -141,6 +141,11 @@ func (ew *eventWorker) writeEvent(e event.IEventStruct) {
 // 解析类型，输出
 func (ew *eventWorker) parserEvents() []byte {
 	ew.status = ProcessStateProcessing
+	tsize := int(ew.processor.truncateSize)
+	if tsize > 0 && ew.payload.Len() > tsize {
+		ew.payload.Truncate(tsize)
```

Comment:
1. Is it just judging the size when parsing the first package? A more suitable solution should be when receiving each Event? For example https://github.com/gojue/ecapture/blob/dcfc3cf1a78e5c1eeb6172ffa16e2be64ebc5c46/pkg/event_processor/iworker.go#L138
2. Is it the most reasonable solution to discard it directly when the payload is greater than `tsize`? Is it more appropriate to continue receiving after printing out?